src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        System.out.println(manufacturerService.get(16L));
+        manufacturerService.delete(16L);
```

Comment:
If you write ids yourself, but do not take them from the database, someone else may have an error trying to launch your project

Reply:
yeah, I just wanted to delete this row while testing  =) 
i'll change it rn